### DISCUSSION TITLE
How to install lensfun in GIMP 2.10.36 under Mint 21.2

### DISCUSSION THREAD

**Discussion by LateJunction** [lensfun/lensfun#2137] [General]
I have understood enough of the contents of ReadMD to build lensfun 0.3.0 from source with the resultant xml's being installed in /usr/local/share/lensfun. I also understand that GIMP 2.10.36 will look for user-installed plugins in ~/.config/GIMP/2.10/plugins, so my  installed version of GIMP does not see the lensfun data. How do I fix this?

**Comment by tuxfanx** [comment]
Why do you want to build lensfun yourself?
I don't use Linux Mint but Debian. But I think that the Mint distribution also contains the lensfun libraries and tools. So why don't you just install these packages? You can then use the lensfun-update-data tool to keep the lensfun database up to date.
